src/main/scala/verifier/WorkerVerifier.scala

```diff
@@ -7,6 +7,7 @@
 package viper.silicon.verifier
 
 import viper.silicon.supporters._
+import viper.silver.ast.Member
```

Comment:
To avoid confusing Silver AST nodes with Silicon terms nodes, always use AST members as `ast.Foo` in contexts where AST nodes and terms might both be used.